src/samples/tester.cpp

```diff
@@ -0,0 +1,232 @@
+// Copyright 2017 Tunnler Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#include <deque>
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+
+#include "tunnler/room.h"
+#include "tunnler/room_member.h"
+
+/**
+ * This will test the functions of the tunnler lib. 
+ * It will create a room listening on 127.0.0.1:1234 and two members connecting to that room.
+ * The output should look like:
+ *
+ * Created a test room
+ *
+ * Joining Member A
+ * Successfully joined Member A
+ * Member A 00:1f:32:7d:3a:3b:
+ *
+ * Joining Member B
+ * Successfully joined Member B
+ * Member A 00:1f:32:7d:3a:3b:
+ * Member B 00:1f:32:18:21:27:
+ * Test room(2/10)          *For now this will look like (2/0) since room_information.name and room_information.number_slots is never set*
```

Comment:
Mention such things in review / issues, not in code comments
We'll forget about it and have confusing comments / partially working code and nobody knows what's correct and what's outdated